src/Config/InvalidMosaicConfigException.ts

```diff
@@ -0,0 +1,7 @@
+export default class InvalidMosaicConfigException extends Error {
```

Comment:
First questions, do we really need individual exceptions? Are we handling/catching them differently? If yes, let's keep them, however, instead of having for each exception a different file, I would propose to have one file (exception.ts) for all exceptions with multiple exports.

Reply:
Custom exception adds more readability about application behavior. 
In this context, we are not handling them but just logging for the user to know what went wrong. In this case, adding custom exception gives the user more info about the kind of error.   Below show how an error is shown to the user: 
```
error: error while executing mosaic create: InvalidMosaicConfigException: requires property "ostComposerAddress" {"service":"mosaic-chains","type":"InvalidMosaicConfigException","timestamp":"2019-06-13T06:35:21.491Z"}
```

https://github.com/OpenST/mosaic-chains/blob/6324ab0127b3942447438eec517824719e40c4c4/src/bin/mosaic-create.ts#L32

What do you think?